src/main/java/net/snowflake/client/core/SFBaseSession.java

```diff
@@ -473,25 +480,49 @@ public HttpClientSettingsKey getHttpClientKey() throws SnowflakeSQLException {
                   "http",
                   userAgentSuffix,
                   gzipDisabled);
+          logDriverInitInfo(ocspAndProxyAndGzipKey);
         } else {
           // Not enough parameters set to use the proxy.
-          logger.debug(
-              "http.useProxy={} but valid host and port were not provided. No proxy in use.",
+          logger.warn(
+              "Failed parsing the proxy settings from JVM parameters as http.useProxy={},"
+                  + " but valid host and port were not provided.",
               httpUseProxy);
           ocspAndProxyAndGzipKey =
-              new HttpClientSettingsKey(getOCSPMode(), userAgentSuffix, gzipDisabled);
+              new HttpClientSettingsKey(ocspMode, userAgentSuffix, gzipDisabled);
+          logDriverInitInfo(ocspAndProxyAndGzipKey);
         }
       } else {
         // If no proxy is used or JVM http proxy is used, no need for setting parameters
         logger.debug("http.useProxy={}. JVM proxy not used.", httpUseProxy);
         unsetInvalidProxyHostAndPort();
-        ocspAndProxyAndGzipKey =
-            new HttpClientSettingsKey(getOCSPMode(), userAgentSuffix, gzipDisabled);
+        ocspAndProxyAndGzipKey = new HttpClientSettingsKey(ocspMode, userAgentSuffix, gzipDisabled);
+        logDriverInitInfo(ocspAndProxyAndGzipKey);
       }
     }
     return ocspAndProxyAndGzipKey;
   }
 
+  private void logDriverInitInfo(HttpClientSettingsKey key) {
```

Comment:
it's driver or http client init info?

Reply:
Right, changed to `logHttpClientInitInfo`.